src/features/firstLogin/PrivacyPolicy/PrivacyPolicy.tsx

```diff
@@ -24,10 +24,16 @@ export function PrivacyPolicy(props: Props) {
     storage.saveObject('has_accepted_cookie', true)
   }
 
+  function refuseCookie() {
+    setHasUserMadeCookieChoice(true)
+    storage.saveObject('has_accepted_cookie', false)
+  }
+
   return hasUserMadeCookieChoice ? null : (
     <PrivacyPolicyModal
       visible={true}
-      dismissModal={acceptCookie}
+      approveModal={acceptCookie}
+      refuseModal={refuseCookie}
```

Comment:
Je challenge le naming qui inclut `Modal` :
```suggestion
      onApproval={acceptCookie}
      onRefusal={refuseCookie}
```

Reply:
OK